app/controllers/users_controller.rb

```diff
@@ -22,6 +24,23 @@ def create
     end
   end
 
+  # GET /users/edit
```

Comment:
Correct

これもパスが間違ってる。
ルーティング確認した？

```suggestion
  # GET /users/:id/edit
```

Reply:
[3029a1a](https://github.com/ok-os-job-change-team/tetsuya-twitter-clone-bootcamp/pull/36/commits/3029a1a61bab4a305274757367b1c806d74260bc) で対応しました。